bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -172,6 +173,54 @@ impl From<matrix_sdk::TransmissionProgress> for TransmissionProgress {
     }
 }
 
+/// An object encapsulating the SSO login flow
```

Comment:
Might be nice to have this in `authentication.rs` too perhaps?

Reply:
Have moved it in b8f69b0ed98942a4b7ca09a38f97d0d4a157e584.